spec/hash-map.dd

```diff
@@ -361,9 +361,13 @@ $(SPEC_RUNNABLE_EXAMPLE_RUN
 
 $(H2 $(LNAME2 static_initialization, Static Initialization of AAs))
 
-        NOTE: Not yet implemented.
+        NOTE: Static initialization, e.g. assigning a global variable with some
```

Comment:
I think the wording is not great, I propose that instead

"Initialization using an AA literal, computed during compilation, is not implemented, instead initialization must be done at run-time, for example in a static constructor."

Can be amended ;)

but otherwise I think that another problem is that this should be a specification of the array literal, not of the AA, i.e the restriction should be in https://dlang.org/spec/expression.html#associative_array_literals.

Reply:
it's also other things than just AA literals, any AA value generated at compile time.

Reply:
improved the wording.